cmd/vic-machine/common/proxy_test.go

```diff
@@ -39,21 +40,29 @@ func TestProcessProxies(t *testing.T) {
 		"httpd://example.com",
 	}
 
+	gnproxies := [...]string{
+		"*",
```

Comment:
Is this actually a valid value? If so, perhaps it should be mentioned in the help text (which currently says "This should be a comma-separated list of hostnames, domain names, or a mixture of both"),

Reply:
Good catch. Just updated the information.